CI still doesn't like me:

```
The following tests FAILED:

	 16 - Whitespace (Failed)

Errors while running CTest

Makefile:130: recipe for target 'test' failed

make[1]: *** [test] Error 8

make[1]: Leaving directory '/build/varlibjenkinsworkspaceubportsunityPRunityubportsgbpbdsc/obj-x86_64-linux-gnu'

dh_auto_test: make -j12 test ARGS+=-j12 returned exit code 2

debian/rules:12: recipe for target 'build-arch' failed

make: *** [build-arch] Error 2

dpkg-buildpackage: error: debian/rules build-arch gave error exit status 2

I: copying local configuration

E: Failed autobuilding of package

I: user script /var/cache/pbuilder/build/cow.1356/tmp/hooks/C10shell starting

  ┌──────────────────────────────────────────────────────────────────────────────┐

  │  FTBFS - problem with building Debian package                                │

  └──────────────────────────────────────────────────────────────────────────────┘

  ┌──────────────────────────────────────────────────────────────────────────────┐

  │  DEB_KEEP_BUILD_ENV is not set to 'true', not keeping build environment      │

  └──────────────────────────────────────────────────────────────────────────────┘

```

--- a/qml/Launcher/Drawer.qml
+++ b/qml/Launcher/Drawer.qml
@@ -74,7 +74,7 @@ FocusScope {
         anchors.fill: parent
         color: "#111111"
         opacity: 0.99
-        
+
         Image {
             id: background
             anchors.fill: parent

